daemon/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "skipLibCheck": false,
```

Comment:
Why?

Reply:
<img width="1356" alt="Screenshot 2024-07-26 at 21 10 03" src="https://github.com/user-attachments/assets/4858afd1-606d-402f-95dd-8bdf863bed6b">

It will cause many dependencies to be unable to introduce other dependencies that are not introduced in the current project.

Reply:
But the problem is that these packages are imported by other packages in `node_modules`, not by MCSManager. I can't find the reason for the issue.

Reply:
the same. they use indirect deps.